bundle/Resources/views/ContentType/field_types.html.twig

```diff
@@ -183,18 +183,34 @@
         {{- form_widget(form.isMultiple) -}}
     </div>
 
-    <div class="ezselection-settings options{% if group_class is not empty %} {{ group_class }}{% endif %}">
+    <div id="{{form.options.vars.id}}-ezselection-root-options-list"
+         class="ezselection-settings options{% if group_class is not empty %} {{ group_class }}{% endif %}"
+         data-prototype="{{ ( form_widget(form.options.vars.prototype) ~ ' ' ~
```

Comment:
I'm talking about the concept. I need to think a bit more about the topic, but instead of storing the *prototype* into a data attribute, I would rather store it in a `<script>` tag and call that a template. Also, we could maybe replace the ugly `String.replace(__whatever__)` by a proper template usage.